config/v3_0/types/config.go

```diff
@@ -15,7 +15,12 @@
 package types
 
 import (
+	"github.com/coreos/ignition/v2/config/util"
+
 	"github.com/coreos/go-semver/semver"
+	"github.com/coreos/ignition/v2/config/shared/errors"
```

Comment:
We still have `coreos/ignition` imports spread across two blocks here.

Reply:
Oops, oversight.